contracts/libraries/VaultLifecycle.sol

```diff
@@ -178,16 +157,13 @@ library VaultLifecycle {
             (controller.getAccountVaultCounter(address(this))).add(1);
 
         IOtoken oToken = IOtoken(oTokenAddress);
-        uint256 strikePrice = oToken.strikePrice();
-        bool isPut = oToken.isPut();
```

Comment:
we are we removing these variables?

Reply:
Getting stack too deep error and it's only used once so I just called it directly below `otoken.isPut()`